doc/modules/calibration.rst

```diff
@@ -205,3 +205,115 @@ a similar decrease in log-loss.
     .. [5] On the combination of forecast probabilities for
            consecutive precipitation periods. Wea. Forecasting, 5, 640–650.,
            Wilks, D. S., 1990a
+
+
+.. _decision_threshold_calibration:
+
+==============================
+Decision Threshold calibration
+==============================
+
+.. currentmodule:: sklearn.calibration
+
+:class:`CutoffClassifier` Often Machine Learning classifiers base their predictions on real-valued decision
```

Comment:
We usually try keep lines under 80 chars

Reply:
uh, didn't think it applies because the rest of the text on the file didn't follow it but no biggie I will fix that.